deepseek:
  model_url: /home/orangepi/Documents/deepseek_weights/DeepSeek-R1-Distill-Qwen-1.5B_W8A8_RK3588.rkllm
  # model_url: /home/orangepi/Documents/deepseek_weights/DeepSeek-R1-Distill-Qwen-1.5B_FP16_RK3588.rkllm
  max_new_tokens: 2048
  max_context_len: 4096
  temperature: 0.8
  top_p: 0.95
  top_k: 1
  repeat_penalty: 1.1
